fmriprep/workflows/bold/resampling.py

```diff
@@ -334,6 +351,40 @@ def _aslist(in_value):
     else:
         mask_mni_tfm.inputs.reference_image = template_out_grid
         bold_to_mni_transform.inputs.reference_image = template_out_grid
+        
+    if freesurfer:
+    # Sample the parcellation files to functional space
+        aseg_mni_tfm = pe.Node(
+            ApplyTransforms(interpolation='MultiLabel', float=True),
+            name='aseg_mni_tfm', mem_gb=1)
+        aparc_mni_tfm = pe.Node(
+            ApplyTransforms(interpolation='MultiLabel', float=True),
+            name='aparc_mni_tfm', mem_gb=1)
+            
+        workflow.connect([
+            (inputnode, aseg_mni_tfm, [('bold_aseg', 'input_image'),('t1_2_mni_forward_transform', 'transforms')]),
+            (inputnode, aparc_mni_tfm, [('bold_aparc', 'input_image'),('t1_2_mni_forward_transform', 'transforms')]),
```

Comment:
Commas should have spaces after them. (In this and the above line, `),(` should be `), (`.)

Reply:
thanks! fixed